### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by tgalopin**
```
- Updating doctrine/cache (v1.6.1 => v1.7.0): Downloading (100%)         
- Updating doctrine/collections (v1.4.0 => v1.5.0): Downloading (100%)         
- Updating doctrine/inflector (v1.1.0 => v1.2.0): Downloading (100%)         
- Updating doctrine/annotations (v1.4.0 => v1.5.0): Downloading (100%)         
- Updating doctrine/common (v2.7.2 => v2.7.3): Downloading (100%)         
- Updating doctrine/dbal (v2.5.12 => v2.5.13): Downloading (100%)         
- Updating sentry/sentry (1.7.0 => 1.7.1): Downloading (100%)         
- Updating ramsey/uuid (3.6.1 => 3.7.0): Downloading (100%)         
- Updating symfony/symfony (v3.3.2 => v3.3.6): Downloading (100%)         
- Updating ramsey/uuid-doctrine (1.3.0 => 1.4.1): Downloading (100%)         
- Updating league/commonmark (0.15.4 => 0.15.6): Downloading (100%)         
- Updating league/flysystem (1.0.40 => 1.0.41): Downloading (100%)         
- Updating guzzlehttp/guzzle (6.2.3 => 6.3.0): Downloading (100%)         
- Updating csa/guzzle-bundle (v2.2.0 => v2.2.1): Downloading (100%)         
- Updating sonata-project/admin-bundle (3.19.0 => 3.20.1): Downloading (100%)         
- Updating sonata-project/doctrine-orm-admin-bundle (3.1.5 => 3.1.6): Downloading (100%)         
- Updating zendframework/zend-eventmanager (3.1.0 => 3.2.0): Downloading (100%)         
- Updating zendframework/zend-code (3.1.0 => 3.2.0): Downloading (100%)         
- Updating scheb/two-factor-bundle (v2.9.0 => v2.11.0): Downloading (100%)         
- Updating willdurand/geocoder-bundle (4.1.0 => 4.2.0): Downloading (100%)         
- Updating knplabs/knp-time-bundle (1.7.3 => 1.7.4): Downloading (100%)         
- Updating suin/php-rss-writer (1.5.0 => 1.6.0): Downloading (100%)         
- Updating facebook/graph-sdk (5.5.0 => 5.6.0): Downloading (100%)         
- Updating maxmind/web-service-common (v0.3.1 => v0.4.0): Downloading (100%)         
- Updating geoip2/geoip2 (v2.5.0 => v2.6.0): Downloading (100%)         
- Updating lullabot/amp (1.1.1 => 1.1.2): Downloading (100%)         
- Updating sensio/generator-bundle (v3.1.4 => v3.1.6): Downloading (100%)         
- Updating symfony/phpunit-bridge (v3.3.2 => v3.3.6): Downloading (100%)         
- Updating sebastian/object-enumerator (3.0.2 => 3.0.3): Downloading (100%)         
- Updating sebastian/environment (3.0.4 => 3.1.0): Downloading (100%)         
- Updating phpunit/phpunit-mock-objects (4.0.1 => 4.0.4): Downloading (100%)         
- Updating phpunit/php-token-stream (1.4.11 => 2.0.0): Downloading (100%)         
- Updating phpunit/php-code-coverage (5.2.1 => 5.2.2): Downloading (100%)         
- Updating phpunit/phpunit (6.2.2 => 6.2.4): Downloading (100%)         
- Updating giggsey/libphonenumber-for-php (8.5.2 => 8.7.1): Downloading (100%)         
- Updating sensiolabs/security-checker (v4.0.4 => v4.1.3): Downloading (100%)         
- Updating gedmo/doctrine-extensions (v2.4.29 => v2.4.30): Downloading (100%)         
- Updating intervention/image (2.3.13 => 2.4.0): Downloading (100%)
```

### CODE REVIEW

**Comment by HeahDude** [comment]
The update was 3 months old. I've launched it again before we merge:
```
  - Updating ocramius/package-versions (1.1.2 => 1.1.3)
  - Updating doctrine/cache (v1.6.1 => v1.7.1)
  - Updating doctrine/inflector (v1.1.0 => v1.2.0)
  - Updating symfony/polyfill-mbstring (v1.4.0 => v1.6.0)
  - Updating twig/twig (v2.4.3 => v2.4.4)     
  - Updating symfony/polyfill-util (v1.4.0 => v1.6.0)     
  - Updating paragonie/random_compat (v2.0.10 => v2.0.11)     
  - Updating symfony/polyfill-php70 (v1.4.0 => v1.6.0)    
  - Updating symfony/polyfill-php56 (v1.4.0 => v1.6.0)
  - Updating symfony/polyfill-intl-icu (v1.4.0 => v1.6.0)   
  - Updating symfony/symfony (v3.3.3 => v3.3.10)
  - Updating doctrine/collections (v1.4.0 => v1.5.0)
  - Updating doctrine/annotations (v1.4.0 => v1.5.0)
  - Updating doctrine/common (v2.7.2 => v2.8.1)
  - Updating symfony/polyfill-apcu (v1.4.0 => v1.6.0)      
  - Updating doctrine/doctrine-cache-bundle (1.3.0 => 1.3.2)  
  - Updating doctrine/dbal (v2.5.12 => v2.6.2)  
  - Updating doctrine/doctrine-bundle (1.6.8 => 1.7.2)     
  - Updating doctrine/doctrine-fixtures-bundle (2.3.0 => v2.4.1)     
  - Updating symfony/monolog-bundle (v3.1.0 => v3.1.1)
  - Updating composer/ca-bundle (1.0.7 => 1.0.8)
  - Updating sensiolabs/security-checker (v4.0.4 => v4.1.6)      
  - Updating sensio/distribution-bundle (v5.0.20 => v5.0.21)
  - Updating fzaninotto/faker (v1.6.0 => v1.7.1) 
  - Updating sentry/sentry (1.7.0 => 1.8.0)  
  - Updating ramsey/uuid (3.6.1 => 3.7.1)
  - Updating doctrine/orm (v2.5.6 => v2.5.12)       
  - Updating ramsey/uuid-doctrine (1.3.0 => 1.4.1)   
  - Updating zendframework/zend-eventmanager (3.1.0 => 3.2.0)  
  - Updating zendframework/zend-code (3.1.0 => 3.3.0)
  - Updating doctrine/doctrine-migrations-bundle (v1.2.1 => v1.3.0)
  - Updating league/commonmark (0.15.4 => 0.15.7)
  - Updating league/flysystem (1.0.40 => 1.0.41)
  - Updating csa/guzzle-bundle (v2.2.0 => v2.2.1)
  - Updating sonata-project/core-bundle (3.4.0 => 3.5.1)       
  - Updating sonata-project/cache (1.0.7 => 2.0.0) 
  - Updating sonata-project/block-bundle (3.3.2 => 3.5.0)    
  - Updating sonata-project/admin-bundle (3.20.1 => 3.24.0)     
  - Updating sonata-project/doctrine-orm-admin-bundle (3.1.5 => 3.1.7)     
  - Updating scheb/two-factor-bundle (v2.9.0 => v2.13.0)       
  - Updating zendframework/zend-diactoros (1.4.0 => 1.6.1) 
  - Updating willdurand/geocoder-bundle (4.1.0 => 4.2.0)
  - Updating snc/redis-bundle (2.0.2 => 2.0.4)
  - Updating beberlei/doctrineextensions (v1.0.17 => v1.0.19)  
  - Updating knplabs/knp-time-bundle (1.7.3 => 1.7.5)       
  - Updating kriswallsmith/buzz (v0.15 => v0.15.1)     
  - Updating giggsey/libphonenumber-for-php (8.5.2 => 8.8.5)      
  - Updating php-amqplib/php-amqplib (v2.6.3 => v2.7.0)
  - Updating sabre/vobject (4.1.2 => 4.1.3)      
  - Updating gedmo/doctrine-extensions (v2.4.30 => v2.4.31)
  - Updating suin/php-rss-writer (1.5.0 => 1.6.0)
  - Updating facebook/graph-sdk (5.5.0 => 5.6.1)    
  - Updating maxmind/web-service-common (v0.3.1 => v0.4.0)     
  - Updating maxmind-db/reader (v1.1.3 => v1.2.0)       
  - Updating geoip2/geoip2 (v2.5.0 => v2.7.0)     
  - Updating masterminds/html5 (2.2.2 => 2.3.0)       
  - Updating marc1706/fast-image-size (v1.1.3 => v1.1.4)        
  - Updating lullabot/amp (1.1.1 => 1.1.2)    
  - Updating sensio/generator-bundle (v3.1.4 => v3.1.6)
  - Updating symfony/phpunit-bridge (v3.3.3 => v3.3.10)         
  - Updating sebastian/object-enumerator (3.0.2 => 3.0.3)
  - Updating phpunit/phpunit-mock-objects (4.0.2 => 4.0.4)
  - Updating phpunit/php-token-stream (1.4.11 => 2.0.1)     
  - Updating phpunit/php-code-coverage (5.2.1 => 5.2.2)
  - Updating phpdocumentor/reflection-common (1.0 => 1.0.1)      
  - Updating phpdocumentor/type-resolver (0.2.1 => 0.4.0)       
  - Updating phpdocumentor/reflection-docblock (3.1.1 => 4.1.1)    
  - Updating phpspec/prophecy (v1.7.0 => v1.7.2)         
  - Updating myclabs/deep-copy (1.6.1 => 1.7.0)       
  - Updating phpunit/phpunit (6.2.3 => 6.2.4)       
  - Updating algolia/algoliasearch-client-php (1.18.0 => 1.23.0)        
  - Updating intervention/image (2.3.14 => 2.4.1)
```

**Comment by HeahDude** [comment]
Tests are failing now, I'm working on it.

**Comment by tgalopin** [comment]
Don't hesitate to merge following your own process now :) .

File: tests/Controller/EnMarche/TonMacronControllerTest.php [hunk 1]
```diff
@@ -226,10 +226,12 @@ private function getChoice(int $id): ?TonMacronChoice
     /**
      * @param int[] $ids
      *
-     * @return TonMacronChoice[]|ArrayCollection
+     * @return TonMacronChoice[]|ArrayCollection|array
      */
-    private function getChoices(array $ids): ArrayCollection
+    private function getChoices(array $ids, bool $asCollection = false)
```
Comment (Remg) [inline tests/Controller/EnMarche/TonMacronControllerTest.php]: no return type?

Comment (HeahDude) [reply tests/Controller/EnMarche/TonMacronControllerTest.php]: `:mixed` is not allowed ;).

Comment (HeahDude) [reply tests/Controller/EnMarche/TonMacronControllerTest.php]: Arf I guess I could use `iterable`.

Comment (HeahDude) [reply tests/Controller/EnMarche/TonMacronControllerTest.php]: Done.

**Pull request merged**